docs/manuals/core/config/data-node-config.md

```diff
@@ -739,10 +739,10 @@ can be provided:
   More optional parameters can be passed through the _**write_fct_params**_ parameter.
 
 - _**read_fct_params**_ represents the parameters passed to the _read_fct_ to
-  read/de-serialize the data. It must be a `List` type object.
+  read/de-serialize the data. It must be a `Tuple` type object.
```

Comment:
```suggestion
  read/de-serialize the data. It must be a `List` type object.
```
This has been changed back to List for simplicity on the user's understanding :D

Reply:
On taipy 2.2, we only allow Tuple. List will be for 2.3